### DISCUSSION TITLE
Rowy Run is failing with error message

### DISCUSSION THREAD

**Discussion by devth** [AntlerVC/firetable#774] [Support / Q&A] [answered]
👋🏻 I installed Rowy in my dev and prod GCP accounts late last year (December 2021, if I remember right). I'm now trying to upgrade to the latest version.

I went to https://docs.rowy.io/setup/install and clicked the "Deploy now" button.

I selected my dev GCP project, where an old version of Rowy was installed, chose the region, then let it run. It eventually errored out with this message:

```
│ Error: resource is in failed state "Ready:False", message: Revision rowy-hooks-8fgzm is not ready.
│
│   with google_cloud_run_service.rowy_hooks,
│   on rowy_hooks.tf line 8, in resource "google_cloud_run_service" "rowy_hooks":
│    8: resource "google_cloud_run_service" "rowy_hooks" {
│
╵
[ ! ] FYI, running the following command:
        docker push gcr.io/my-gcp-project/rowy-builder
[ ✖ ] Failed to push container image to Google Container Registry.
Error: failed to push image to gcr.io/my-gcp-project/rowy-builder: docker push failed: exit status 1, output:
Using default tag: latest
The push refers to repository [gcr.io/my-gcp-project/rowy-builder]
An image does not exist locally with the tag: gcr.io/my-gcp-project/rowy-builder
```

 I tried the same in my prod GCP project, which also had a previous version of Rowy installed and encountered the same error.

1. Do I need to uninstall the existing version of Rowy before I reinstall?
2. How do I uninstall Rowy? I didn't see instructions.

Thanks 🙏🏻

**Comment by harinij** [comment, marked as answer]
Hi @devth, apologies we missed this message. Are you still looking for a solution? Have you tried the [Rowy ap](https://rowy.app/) it has a guided process for you to get setup or alternatively if it self hosted if you can share a bit more about the version you are on, it would be helpful to further assist.

**Reply by devth** [reply]
No worries I started using the hosted app instead. Thanks!
